src/main/java/life/mosu/mosuserver/presentation/oauth/OAuthController.java

```diff
@@ -0,0 +1,25 @@
+package life.mosu.mosuserver.presentation.oauth;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.view.RedirectView;
+
+import static life.mosu.mosuserver.presentation.oauth.AuthorizationRequestRedirectResolver.REDIRECT_PARAM_KEY;
+
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/auth")
+public class OAuthController {
+
+    @GetMapping("/login/{registrationId}")
+    public RedirectView login(@PathVariable String registrationId, @RequestParam(REDIRECT_PARAM_KEY) String redirect) {
+        final String url = String.format(
+            "/oauth2/authorization/%s?%s=%s",
+            registrationId,
+            REDIRECT_PARAM_KEY,
+            redirect
+        );
+        return new RedirectView(url);
```

Comment:
```suggestion
    final String url = UriComponentsBuilder
        .fromPath("/oauth2/authorization/{registrationId}")
        .queryParam(REDIRECT_PARAM_KEY, redirect)
        .buildAndExpand(registrationId)
        .toUriString();
    return new RedirectView(url);
```
`String.format` 메소드는 안정성이 높은 메소드는 아닙니다. 해당 부분으로 변경하는건 어떨까요?